front/lib/models/assistant/agent.ts

```diff
@@ -262,6 +262,7 @@ export class AgentUserRelation extends Model<
   declare agentConfiguration: string;
 
   declare listStatusOverride: AgentUserListStatus | null;
```

Comment:
True the PR was unclear => the column is removed in a follow-up PR (mentioned at the end of the description) since we can't add+ delete cols in the same PR (notably for backfill scripts builds)
It's indeed here more adding a column + removing another rather than 'renaming' I agree

Reply:
ok!